### ISSUE TITLE
App Outdated screen not being shown

### ISSUE THREAD

**Issue by khoi** [toggl/mobileapp#3674]
## 🐛 Bug Report
When the app is outdated, it doesn't show the Outdated screen.

## Context

Build/Branch: `develop`
Phone OS: Droid
Phone model: Any Model

## Expected behavior
The screen should be shown and blocking the user from using the outdated app.

## To reproduce
The easiest way to replicate an outdated state is by hardcoding

In `App.cs` add `|| true` to `NavigateToFirstViewModel`

```
if (true || accessRestrictionStorage.IsApiOutdated() || accessRestrictionStorage.IsClientOutdated())
```

**Comment by heytherewill** [comment]
This is fixed, isn't it, @daividssilverio ?

**Comment by daividssilverio** [comment]
Yeah, fixed by #3951.
